joomla/joomla-cms
# Reinit mediahelper test

### Summary of Changes
I had to disable the Tests for the JMediaHelper because I found out that test fail just a few minutes before release. This PR enables that tests. Because we didn't really tested the mine type we hadn't the problem with how we load data stubs for the extensions table. We define minetypes 

```
image\\/jpeg  
```

The double Backslash results in a check 

```
"image/jpeg" === "image\/jpeg"
```

Obviously not the same and the test is failing.

### Testing Instructions
To see it fail go to https://github.com/joomla/joomla-cms/blob/staging/tests/unit/suites/libraries/cms/helper/JHelperMediaTest.php#L183 and delete the line 

the let the test run (do a composer install in your repository dir and then ./libraries/vendor/bin/phpunit tests/unit/suites/libraries/cms/helper/JHelperMediaTest.php)

* Apply Patch
* Run the test again, test should not fail
* Run all tests, no test should fail
